docs/sources/getting-started/glossary.md

```diff
@@ -17,41 +17,72 @@ This topic lists words and abbreviations that are commonly used in the Grafana d
 
 <table>
   <tr>
-    <td style = "vertical-align: top;">Dashboard</td>
-    <td>A set of one or more panels, organized and arranged into one or more rows, that provide an at-a-glance view of related information.</td>
+    <td style="vertical-align: top">Dashboard</td>
+    <td>
+      A set of one or more panels, organized and arranged into one or more rows, that provide an at-a-glance view of related information.
+    </td>
+  </tr>
+  <tr>
+    <td style="vertical-align: top">Data source</td>
+    <td>
+      A file, database, or service providing the data. Grafana supports a several data sources by default, and can be extended to support additional ones through plugins.
+    </td>
   </tr>
   <tr>
-    <td style = "vertical-align: top;">Data source</td>
-    <td>A file, database, or service providing the data. Grafana supports a several data sources by default, and can be extended to support additional ones through plugins.</td>
+    <td style="vertical-align: top">Exemplar</td>
+    <td>
+      An exemplar is any data that serves as a detailed example of data used to create an aggregation. A common type of examplar is a trace that references an aggregated metric.
+    </td>
+  </tr>
+  <tr>
+    <td style="vertical-align: top">Graph</td>
+    <td>
+      A commonly-used visualization that displays data as points, lines, or bars.
+    </td>
   </tr>
-    <tr>
-    <td style = "vertical-align: top;">Graph</td>
-    <td>A commonly-used visualization that displays data as points, lines, or bars.</td>
+  <tr>
+    <td style="vertical-align: top">Panel</td>
+    <td>
+      Basic building block in Grafana, composed by a query and a visualization. Can be moved and resized within a dashboard.
+    </td>
   </tr>
-    <tr>
-    <td style = "vertical-align: top;">Panel</td>
-    <td>Basic building block in Grafana, composed by a query and a visualization. Can be moved and resized within a dashboard.</td>
+  <tr>
+    <td style="vertical-align: top">Plugin</td>
+    <td>
+      An extension of Grafana that allows users to provide additional functionality to enhance their experience. The types of plugins currently supported are:
```

Comment:
Don't we have these defined somewhere else? It might be more efficient to link to the Plugins concept page. That way we don't have to maintain two lists. I'd say keep the first sentence, then say, "For more information, refer to LINK."

Reply:
I'll fix this in a separate PR!